test/plugin/scenarios/feign-scenario/support-version.list

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+9.0.0
+9.1.0
+9.4.0
+9.5.0
```

Comment:
are you missing some versions?

Reply:
These missing versions of the jar package can not be downloaded from nexus.

Reply:
If so, I think it can be downloaded in our Jenkins VM. Please, try it.

Reply:
@aderm What versions are missing here? Are they available at Maven central? I think that is the key

Reply:
I am using Alibaba Cloud's warehouse. He is not necessarily full here. The local test cannot pass, so I am updating the list now. Maven central should have it.